### ISSUE TITLE
StepLr

### ISSUE THREAD

**Issue by vodanhbk95** [yeyupiaoling/AudioClassification-Pytorch#2]
in your code, you're using steplr `scheduler = StepLR(optimizer, step_size=args.learning_rate, gamma=0.8, verbose=True)` . It's mean that you're using step_size=1e-3. Something wrong here???

**Comment by yeyupiaoling** [comment]
Yes, I have fixed it. Thank you for bringing it up.
